scripts/extensions.sh

```diff
@@ -57,7 +59,7 @@ echo -e "\n\n## meza: Install composer-supported extensions"
 cd "$m_mediawiki"
 cmd_profile "START extensions composer require"
 composer require \
-	mediawiki/semantic-media-wiki:~2.0 \
+	mediawiki/semantic-media-wiki:dev-master#787d70fb7f1c1db800e02e0dde0eee8ccc4b7e95 \
```

Comment:
Can't checkout 2.4 because it's not released yet. We'd have to be running on SMW's master branch, which is a moving target. Regarding why we need this commit: I did a pretty crappy job documenting, didn't I? I've updated #362 with a little more info. Basically: `rebuildData.php` fails on some pages without this.